perf_bench_test.go

```diff
@@ -37,114 +41,192 @@ import (
 var (
 	isColdStart     = flag.Bool("coldStart", false, "Profile cold starts (default is false)")
 	vmNum           = flag.Int("vm", 2, "The number of VMs")
-	targetReqPerSec = flag.Int("requestPerSec", 2, "The target number of requests per second")
-	executionTime   = flag.Int("executionTime", 2, "The execution time of the benchmark in seconds")
+	targetReqPerSec = flag.Int("requestPerSec", 10, "The target number of requests per second")
+	executionTime   = flag.Int("executionTime", 1, "The execution time of the benchmark in seconds")
+	funcNames       = flag.String("funcNames", "helloworld", "Name of the functions to benchmark")
+	perfEvents      = flag.String("perfEvents", "", "Perf events (run `perf stat` if not empty)")
+
+	envPath = "PATH=" + os.Getenv("PATH")
```

Comment:
this is definitely wrong. you should not require this.

Reply:
Done